modules/gitops-disable-dex.adoc

```diff
@@ -19,4 +19,9 @@ Dex is installed by default for all the Argo CD instances created by the Operato
     env:
     - name: DISABLE_DEX
       value: "true"
-----
```

Comment:
Also, above, at the beginning of the file, would it help to briefly mention what Dex is?

How about:
```
You can configure {gitops-title} to use Dex as the SSO authentication provider by setting the `.spec.dex` parameter. 

However, because the `.spec.dex` parameter is deprecated in {gitops-title} v1.6.0 and is planned to be removed in v1.9.0, consider using the `.spec.sso.dex` parameter instead. See "Enabling or disabling Dex using .spec.sso."
```